src/SystemServicesContext.tsx

```diff
@@ -0,0 +1,135 @@
+import React, {
+  createContext,
+  useState,
+  useEffect,
+  FunctionComponent,
+  useContext,
+} from "react"
+import { AppState, Platform } from "react-native"
+
+import { isBluetoothEnabled } from "./gaen/nativeModule"
+import { isLocationEnabled } from "./gaen/nativeModule"
+import { doesDeviceSupportLocationlessScanning } from "./gaen/nativeModule"
+
+export interface SystemServicesState {
+  isBluetoothOn: boolean
+  isLocationOn: boolean
```

Comment:
for iOS would this be not known?

so should we have this be more like
```typescript
isLocationOn: boolean | "unknown"
```

Reply:
The context is set up such that this returns `true` for iOS which seems correct to me.

Reply:
Turns out we do know on iOS because `isLocationEnabled()` works cross-platform. So this can just be a `boolean`.